### PR TITLE
Scripts for  "RegisterAppInterface"

### PR DESCRIPTION
**Pull request by HSavynetska**
ATF Test Scripts to check RegisterAppInterface RPC in specific cases for RAI:
 * DUPLICATE_NAME 
* WARNINGS 
* APPLICATION_REGISTERED_ALREADY
* systemSoftwareVersion
* sdl version
* DISALLOWED
* WRONG_LANGUAGE

This PR is **not ready** for review.
This is **draft** version of the script.

### Summary
Scripts for  "RegisterAppInterface"

### ATF version
[Develop branch](https://github.com/smartdevicelink/sdl_atf/tree/develop)

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: test_scripts/API/Registration/012_SystemSoftwareVersion_only_once_use_value_durin_ignition_cycle.lua [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    return params
+end
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", common.preconditions)
+runner.Step("Start SDL, init HMI, connect Mobile", common.start, { getHMIValuesDuringFirstRAI () })
+
+runner.Title("Test")
+runner.Step("SystemSoftwareVersion in RAI response during first registration", common.registerApp)
```
Comment (GetmanetsIrina) [inline test_scripts/API/Registration/012_SystemSoftwareVersion_only_once_use_value_durin_ignition_cycle.lua]: description is related to receiving GetSystemInfo during registration and steps are checked the SystemSoftwareVersion

File: test_scripts/API/Registration/013_GetSystemInfo_does_not_sent_during_Registration.lua [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+-- Regression check
+-- User story:TBD
+-- Use case:TBD
+--
+-- Requirement summary:
+-- TBD
+--
+-- Description:
+-- In case:
+-- 1) Check that SDL returns SDL version in RegisterAppInterface response
```
Comment (GetmanetsIrina) [inline test_scripts/API/Registration/013_GetSystemInfo_does_not_sent_during_Registration.lua]: description is not clear, please update.

File: test_scripts/API/Registration/013_GetSystemInfo_does_not_sent_during_Registration.lua [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/API/Registration/commonRAI')
+local commonFunctions = require("user_modules/shared_testcases/commonFunctions")
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Functions ]]
+local sdlVersionsParams = {
+    string = "SDL_4",
```
Comment (GetmanetsIrina) [inline test_scripts/API/Registration/013_GetSystemInfo_does_not_sent_during_Registration.lua]: please check only for one value

**Comment by dboltovskyi** [comment]
Livio team please do not review this PR for now since it hasn't passed our internal review yet.
Review and appropriate updates going be done till 1st of November.
Thank you.

**Pull request closed without merging**